tendrl/alerting/notification/handlers/mail_handler.py

```diff
@@ -181,15 +181,14 @@ def __init__(self):
             self.admin_config = {}
             self.user_configs = []
             self.complete = multiprocessing.Event()
-            self.set_destinations()
         except (ConfigNotFound, NotificationDispatchError) as ex:
             raise NotificationDispatchError(str(ex))
 
     def get_mail_client(self):
         if not self.admin_config:
             raise NotificationDispatchError(
                 "Admin mail configuration is required for dispatching email"
-                "notification"
+                " notification"
```

Comment:
Yes because, its a string concatenation of 2 lines "Admin mail configuration is required for dispatching email" and "notification" so if its not " notification", the resultant sentence appears as:
"Admin mail configuration is required for dispatching emailnotification"